Running_Linux_OSX.txt

```diff
@@ -43,7 +44,7 @@ The following need to be done at least once. They do not need to be repeated for
 Autopsy depends on a specific version of The Sleuth Kit.  You need the Java libraries of The Sleuth Kit installed, which is not part of all packages.
 
 - Linux: Install the sleuthkit-java.deb file that you can download from github.com/sleuthkit/sleuthkit/releases.  This will install libewf, etc.
--- % sudo apt install ./sleuthkit-java_4.7.0-1_amd64.deb
+-- % sudo dpkg -i sleuthkit-java_4.7.0-1_amd64.deb
```

Comment:
I don't think this should be changed to use dpkg here because (a) dpkg doesn't understand package dependencies and (b) the rest of this document uses APT tools.